package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/server": "^4.7.0",
```

Comment:
https://www.apollographql.com/docs/apollo-server/getting-started
Make sure you're aware of why you're using apollo/server for
You can use Apollo Server as:
> The GraphQL server for a subgraph in a federated supergraph     
> A stand-alone GraphQL server     
> An add-on to your application's existing Node.js middleware (such as Express, AWS Lambda, or Fastify)